test/functional/forall/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+# Copyright (c) 2016-19, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+raja_add_test(
+  NAME test-forall-seq
+  SOURCES test-forall-seq.cpp)
+
+raja_add_test(
+  NAME test-forall-openmp
+  SOURCES test-forall-openmp.cpp)
```

Comment:
@rhornung67 All tests will run successfully if their target architecture is enabled or not. Do we want to still build these tests even when they aren't enabled?

Reply:
I think it's best to only build the tests when the corresponding back-ends are enabled. @davidbeckingsale @MrBurmark @trws do you agree?

Reply:
I agree with @rhornung67, we only want to build them if the backend is there.  I'm vaguely surprised they will build without the appropriate backend, what happens do they just build a null test?

Reply:
Okay brilliant, I just wanted to check. Essentially yes, the tests just never gets instantiated with the specific back-end execution types.